### ISSUE TITLE
fabio:BachelorThesis, fabio:DoctoralThesis, fabio:MasterThesis

### ISSUE THREAD

**Issue by tawahle** [openrif/vivo-isf-ontology#745]
bibo:Thesis appears to be too general. In [FaBiO](https://sparontologies.github.io/fabio/current/fabio.html#d4e2040) fabio:Thesis is differentiated into fabio:BachelorThesis, fabio:MasterThesis, and fabio:DoctoralThesis. How about to extend VIVO with these classes?

This is https://jira.duraspace.org/browse/VIVO-1537

**Comment by mconlon17** [comment]
This is an add.  No impact to existing code or data.  Impact low.

**Comment by tawahle** [comment]
Yes, but some template adjustments are still necessary. Although, it doesn't have to do with ontology.
